### PR TITLE
publish to rubygems

### PR DESCRIPTION
**Pull request by yevsafronov**
This PR adds the configuration to publish lhm to `lhm-shopify` on rubygems

### CODE REVIEW

**Comment by shuhaowu** [comment]
Is the shipit.rubygems.yml file supposed to be empty?

File: lhm.gemspec [hunk 1]
```diff
@@ -6,15 +6,15 @@ $:.unshift(lib) unless $:.include?(lib)
 require 'lhm/version'
```
[the remark this answers is not in the document]
Comment (yevsafronov) [reply lhm.gemspec:6]: probably not, we are not changing any internal `lhm` naming, just the gem name

File: lhm.gemspec [hunk 2]
```diff
@@ -6,15 +6,15 @@ $:.unshift(lib) unless $:.include?(lib)
...
 
 Gem::Specification.new do |s|
-  s.name          = 'lhm'
+  s.name          = 'lhm-shopify'
   s.version       = Lhm::VERSION
   s.licenses      = ['BSD-3-Clause']
   s.platform      = Gem::Platform::RUBY
   s.authors       = ['SoundCloud', 'Rany Keddo', 'Tobias Bielohlawek', 'Tobias Schmidt']
```
[the remark this answers is not in the document]
Comment (yevsafronov) [reply lhm.gemspec]: good point, added

**Review by shuhaowu — APPROVED** [review]
Seems OK to me. Does this still need to be a draft PR?

Also if you release a new version, please tag it as well.

**Pull request merged**